src/compiler/types.ts

```diff
@@ -5626,6 +5626,7 @@ export interface EmitResolver {
     isEntityNameVisible(entityName: EntityNameOrEntityNameExpression, enclosingDeclaration: Node): SymbolVisibilityResult;
     // Returns the constant value this property access resolves to, or 'undefined' for a non-constant
     getConstantValue(node: EnumMember | PropertyAccessExpression | ElementAccessExpression): string | number | undefined;

```

Comment:
Does this still need to have `EnumMember`?

Reply:
Actually, `declarations.ts` still calls `getConstantValue` but only on an enum value, which makes me think that the resolver wouldn't need that func at all anymore. But, `ts.ts`'s `tryGetConstEnumValue` still calls it for property access / element accesses :(